docs/src/behavior.md

```diff
@@ -24,24 +24,34 @@ docsearch({
 
 ## `handleSelected`
 
-This method is called when a suggestion is selected. By default, DocSearch will
-redirect the browser to the results page at the related anchor, but you can
-override it to add your own behavior.
+This method is called when a suggestion is selected (either with a click or
+keyboard). By default, DocSearch will redirect the browser to the results page
+at the related anchor, but you can override it to add your own behavior.
 
-The method is called with three arguments:
+The method is called with the following arguments:
 
 - `input`, a reference to the search `input` element. It comes with the
   `.open()`, `.close()`, `.getVal()` and `.setVal()` methods.
 
-- `event`, the actual event triggering the selection. This can come from a click
-  or a keyboard navigation.
+- `event`, the actual event triggering the selection.
 
-- `suggestion`, the object representing the current selection.
+- `suggestion`, the object representing the current selection. Contains a `.url`
+  key representing the destination.
+
+- `datasetNumber`, this should always equal `1` as DocSearch is searching
```

Comment:
IMHO we should not document this input if it is not useful with our UI which is already tight with one index only.

Reply:
The issue is that it's the 3rd parameter out of 4, that was already there before. If I don't document it, it make it look like there are only 3 parameters. And we need to keep it to not break backward compatibility, so I unfortunately have to keep it that way :/